db/migrate/20200224201131_create_moment_categories_join_table.rb

```diff
@@ -1,12 +1,12 @@
-class CreateMomentCategoriesJoinTable < ActiveRecord::Migration[5.2]
+class CreateMomentStrategiesJoinTable < ActiveRecord::Migration[5.2]
   def change
-    create_table :moments_categories do |t|
+    create_table :moments_strategies do |t|
       t.integer :moment_id
-      t.integer :category_id
+      t.integer :strategy_id
     end
 
-    add_index :moments_categories, [:moment_id, :category_id], unique: true
-    add_foreign_key :moments_categories, :moments
-    add_foreign_key :moments_categories, :categories
+    add_index :moments_strategies, [:moment_id, :strategy_id], unique: true
+    add_foreign_key :moments_strategies, :moments
+    add_foreign_key :moments_strategies, :strategies
```

Comment:
You will need to create an entirely new migration to make these changes since we do not rollback migrations in production.

Reply:
You're getting migration errors in the CircleCI build because of what I pointed out. I recommend reading this [article](https://guides.rubyonrails.org/active_record_migrations.html) about migrations. You'll want to create a new migration for your changes using `rails g migration [etc..]`.

Reply:
So it looks like actually already have a `moments_strategies` table. I'm not sure why we need to override the migration for `moments_categories` with a table that already exists. Looks like we just want to unstaged and remove this change entirely.

Reply:
That was a totally unintentional commit, sorry about that!